deploy/provision.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 
 # Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.
-#
+# -----------------------------------------------------------------------------
 # This program and the accompanying materials are made available under the
 # terms of the Eclipse Public License v1.0 which accompanies this distribution,
 # and is available at http://www.eclipse.org/legal/epl-v10.html
 
 TEST_USER="jenkins"
-#PROXY="http://172.28.40.9:3128"
-VENV_DIR="venv"
+BASE_DIR=/opt
```

Comment:
Be consistent with variables initialization: either use double quotes everywhere, or don't use them at all